src/contexts/addresses.tsx

```diff
@@ -171,13 +171,10 @@ export const AddressesContextProvider: FC<{ overrideContextValue?: PartialDeep<A
     (addressState) => addressState.network === currentNetwork
   )
 
-  // Used for showing flashing on the large summary card
```

Comment:
This is an inaccurate comment. `addressesWithPendingSentTxs` is used for multiple reasons. It's used for defining the receiving addresses with pending transactions so that their info gets refreshed and also used in the calculation of whether data is loading or not.